yarn.lock

```diff
@@ -3289,6 +3289,33 @@
   resolved "https://registry.yarnpkg.com/@redocly/react-dropdown-aria/-/react-dropdown-aria-2.0.11.tgz#532b864b329237e646abe45d0f8edc923e77370a"
   integrity sha512-rmuSC2JFFl4DkPDdGVrmffT9KcbG2AB5jvhxPIrOc1dO9mHRMUUftQY35KZlvWqqSSqVn+AM+J9dhiTo1ZqR8A==
 
+"@requestnetwork/currency@0.2.0":
```

Comment:
This is odd. Let me check something.

Reply:
Found it. We made a publish before I merge my last PR. 

Can you wait the merge of: https://github.com/RequestNetwork/requestNetwork/pull/436.
Then pull the change from master, do a `yarn` and then commit the new `yarn.lock` please.

Reply:
merged done.